### ISSUE TITLE
Is the object detection dataset used in the paper "Abductive Ego-View Accident Video Understanding for Safe Driving Perception" located in the MMAU_det_paper directory? Is it a part of the MMAU dataset? What do V1-Train and V2-Train represent in the paper?

### ISSUE THREAD

**Issue by QY6616** [jeffreychou777/LOTVS-MMAU#6]
Is the object detection dataset used in the paper "Abductive Ego-View Accident Video Understanding for Safe Driving Perception" located in the MMAU_det_paper directory? Is it a part of the MMAU dataset? What do V1-Train and V2-Train represent in the paper?

**Comment by jeffreychou777** [comment]
Yes, we have explained the meaning of "V1-Train" and "V2-Train" in the subsection 5.1 Experiments Details (1) od tasks.

These are two different training scenarios that differ in the data they contain (outside the accident window or in the accident window, identified by different colored squares)

**Comment by QY6616** [comment]
> Yes, we have explained the meaning of "V1-Train" and "V2-Train" in the subsection 5.1 Experiments Details (1) od tasks.
> 
> These are two different training scenarios that differ in the data they contain (outside the accident window or in the accident window, identified by different colored squares)

@jeffreychou777  Thank you for your answer. May I ask how you divide the dataset into accident windows and non-accident windows? After downloading the dataset, I found that there are only train/test/val splits, but no distinction between train_v1 and train_v2.

**Comment by jeffreychou777** [comment]
Hi, 
Apologies for the naming confusion, We use different colored squares to represent different windows in the paper, and text like train-v1 to identify whether or not the training set contains data from the accident window.

The dataset you download in the MMAU_det_paper directory, is a object detection dataset for all data, i.e., all windows are uniformly distributed in the training, test, and validation sets of these two datasets. The difference between these two datasets is that the dataset used in the paper is constructed entirely according to the format of the coco dataset, omitting the information that these photos originally belonged to the MM-AU dataset, whereas MMAU-Detectv1 retains this information for each image.
If you are further dividing the dataset based on incident windows, you can use MMAU-Detectv1 in conjunction with the window labeling in the excel file.

I hope this helps you.

**Comment by QY6616** [comment]
> 你好， 对于命名混乱，我们深表歉意，我们在论文中使用不同颜色的方块来表示不同的窗口，并使用像 train-v1 这样的文本来识别训练集是否包含来自事故窗口的数据。
> 
> 您在 MMAU_det_paper 目录中下载的数据集是一个针对所有数据的目标检测数据集，即所有窗口均匀分布在这两个数据集的训练集、测试集和验证集中。这两个数据集的区别在于，本文中使用的数据集完全按照 coco 数据集的格式构建，忽略了这些照片原本属于 MM-AU 数据集的信息，而 MMAU-Detectv1 则为每张图片保留了这些信息。 如果您要根据事件窗口进一步划分数据集，可以将 MMAU-Detectv1 与 Excel 文件中的窗口标记结合使用。
> 
> 我希望这对你有帮助。

@jeffreychou777  Thank you very much for your response. Based on your answer, my understanding is: The dataset in the MMAU_det_paper directory corresponds to V1_train/val/test in the figure ? If I want to get the V2_train and its corresponding two test datasets, I need to combine MMAU-Detectv1 with the window labels in the Excel file for further extraction? I'm not sure if my understanding is correct?

<img width="194" alt="Image" src="https://github.com/user-attachments/assets/07817cec-3071-4440-9597-67b0081e3c89" />

**Comment by jeffreychou777** [comment]
> > 你好， 对于命名混乱，我们深表歉意，我们在论文中使用不同颜色的方块来表示不同的窗口，并使用像 train-v1 这样的文本来识别训练集是否包含来自事故窗口的数据。
> > 您在 MMAU_det_paper 目录中下载的数据集是一个针对所有数据的目标检测数据集，即所有窗口均匀分布在这两个数据集的训练集、测试集和验证集中。这两个数据集的区别在于，本文中使用的数据集完全按照 coco 数据集的格式构建，忽略了这些照片原本属于 MM-AU 数据集的信息，而 MMAU-Detectv1 则为每张图片保留了这些信息。 如果您要根据事件窗口进一步划分数据集，可以将 MMAU-Detectv1 与 Excel 文件中的窗口标记结合使用。
> > 我希望这对你有帮助。
> 
> [@jeffreychou777](https://github.com/jeffreychou777) Thank you very much for your response. Based on your answer, my understanding is: The dataset in the MMAU_det_paper directory corresponds to V1_train/val/test in the figure ? If I want to get the V2_train and its corresponding two test datasets, I need to combine MMAU-Detectv1 with the window labels in the Excel file for further extraction? I'm not sure if my understanding is correct?
> 
> <img alt="Image" width="194" src="https://private-user-images.githubusercontent.com/126632974/434587416-07817cec-3071-4440-9597-67b0081e3c89.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQ4NjA5MTcsIm5iZiI6MTc0NDg2MDYxNywicGF0aCI6Ii8xMjY2MzI5NzQvNDM0NTg3NDE2LTA3ODE3Y2VjLTMwNzEtNDQ0MC05NTk3LTY3YjAwODFlM2M4OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNDE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDQxN1QwMzMwMTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNGUzMThiNjc2NjY0MDBiOGVhM2YyNzY5ZjI0MTI4ZWQwMGEyYzg3MThiNDliZTExMTcyZmEyY2NlMmQ4NDQzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.xHjm7IoMH8AZ3n2GZeUU0zADWzvjjrzjxwGM6BAVdz0">
Yes, you are right.

**Comment by QY6616** [comment]
Thank you for your reply！ I'll give it a try. Your work is excellent！I'm looking forward to your open-sourcing more resources.



---- 回复的原邮件 ----
| 发件人 | Jeffrey ***@***.***> |
| 日期 | 2025年04月17日 11:33 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [jeffreychou777/LOTVS-MM-AU] Is the object detection dataset used in the paper "Abductive Ego-View Accident Video Understanding for Safe Driving Perception" located in the MMAU_det_paper directory? Is it a part of the MMAU dataset? What do V1-Train and V2-Train represent in the paper? (Issue #6) |

你好， 对于命名混乱，我们深表歉意，我们在论文中使用不同颜色的方块来表示不同的窗口，并使用像 train-v1 这样的文本来识别训练集是否包含来自事故窗口的数据。
您在 MMAU_det_paper 目录中下载的数据集是一个针对所有数据的目标检测数据集，即所有窗口均匀分布在这两个数据集的训练集、测试集和验证集中。这两个数据集的区别在于，本文中使用的数据集完全按照 coco 数据集的格式构建，忽略了这些照片原本属于 MM-AU 数据集的信息，而 MMAU-Detectv1 则为每张图片保留了这些信息。 如果您要根据事件窗口进一步划分数据集，可以将 MMAU-Detectv1 与 Excel 文件中的窗口标记结合使用。
我希望这对你有帮助。

@jeffreychou777 Thank you very much for your response. Based on your answer, my understanding is: The dataset in the MMAU_det_paper directory corresponds to V1_train/val/test in the figure ? If I want to get the V2_train and its corresponding two test datasets, I need to combine MMAU-Detectv1 with the window labels in the Excel file for further extraction? I'm not sure if my understanding is correct?

Yes, you are right.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>

jeffreychou777 left a comment (jeffreychou777/LOTVS-MM-AU#6)

你好， 对于命名混乱，我们深表歉意，我们在论文中使用不同颜色的方块来表示不同的窗口，并使用像 train-v1 这样的文本来识别训练集是否包含来自事故窗口的数据。
您在 MMAU_det_paper 目录中下载的数据集是一个针对所有数据的目标检测数据集，即所有窗口均匀分布在这两个数据集的训练集、测试集和验证集中。这两个数据集的区别在于，本文中使用的数据集完全按照 coco 数据集的格式构建，忽略了这些照片原本属于 MM-AU 数据集的信息，而 MMAU-Detectv1 则为每张图片保留了这些信息。 如果您要根据事件窗口进一步划分数据集，可以将 MMAU-Detectv1 与 Excel 文件中的窗口标记结合使用。
我希望这对你有帮助。

@jeffreychou777 Thank you very much for your response. Based on your answer, my understanding is: The dataset in the MMAU_det_paper directory corresponds to V1_train/val/test in the figure ? If I want to get the V2_train and its corresponding two test datasets, I need to combine MMAU-Detectv1 with the window labels in the Excel file for further extraction? I'm not sure if my understanding is correct?

Yes, you are right.
